### ISSUE TITLE
This isn't an issue, more a request for info / help

### ISSUE THREAD

**Issue by Fliptop72** [dariowouters/ts-fmod-plugin#7]
Hi there, love this plugin, the game is so much better with it, my request for info / help is this:

I have an air brake sound mod that I use, can this be added to this plugin?

I have tried to see if it works myself but have had no joy, any help on this would be much appreciated.

Thanks in advance

**Comment by dariowouters** [comment]
No, the plugin does not support any `truck effects` sounds because the modded sound would play together with the default game sounds unless you mute the whole `truck effects` category in-game, but that would mute a lot of different sounds too like the reverse beep, horn, gearbox sounds, etc.
So to be able to add support for the sound effects, I would have to be able to implement all those sounds which I have not been able to.

And even if I find a way to implement them all, the issue would be sound mods that don't include all the sound effects and so you would be missing sounds as the plugin can't fall back to the default game sounds.

So unless I figure out some way around that and actually have to motivation to work on this plugin other than simply updating it, it unfortunately probably won't be added anytime soon either.
